### DISCUSSION TITLE
Adding in-app tip jar and adding additional payment checks

### DISCUSSION THREAD

**Discussion by amir-fo** [isontheline/pro.webssh.net#933] [Ideas]
Hello Mengus,

It’d be great if there was a tip jar in the app to pay with apple id and not paypal or credit card. Or maybe add your crypto wallet info. 

I have a jailbroken device and in webssh its really easy to bypass the payment and unlock pro features. Its probably a small percentage of your users who have jailbroken devices but you and the app you made are so amazing that i dont like this app and all your efforts to be taken advantage of. So it would be great if you could add a confirmation method to avoid this.

**Comment by isontheline** [comment]
Hello Amir 👋

I understand your point of view but I still trust in users 😇

Everyone can support my work by becoming a sponsor or by buying a coffee : https://webssh.net/documentation/sponsors/

Thank you so much for your feedback Amir 🙏
